packages/flutter/test/material/input_decorator_test.dart

```diff
@@ -1829,6 +1829,33 @@ void main() {
     expect(tester.getTopRight(find.byKey(pKey)).dx, tester.getTopLeft(find.text('text')).dx);
   });
 
+
+  testWidgets('InputDecorator errorBuilder', (WidgetTester tester) async {
+    /// when errorBuilder is defined error message will be from the validator,
```

Comment:
nit: `//` for regular comments.

Reply:
nit: `When errorBuilder is defined, the error message will be from the validator, and the errorMessage will never be null.`?